tests/components/swiss_public_transport/test_config_flow.py

```diff
@@ -0,0 +1,228 @@
+"""Test the swiss_public_transport config flow."""
+from unittest.mock import AsyncMock
+
+import aiohttp
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.swiss_public_transport import config_flow
+from homeassistant.components.swiss_public_transport.const import (
+    CONF_DESTINATION,
+    CONF_START,
+)
+from homeassistant.const import CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+from tests.test_util.aiohttp import AiohttpClientMocker
+
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
+
+
+async def test_flow_user_init(hass: HomeAssistant, aioclient_mock: AiohttpClientMocker):
```

Comment:
You can merge this test with the next one imo

Reply:
I would rather prefer test one branch per unit test, no?

Reply:
I usually just create 1 test for the full config flow (the happy path), the errors in the flow and then the same for the import flow